articles/vmware/vmw-ref-trouble-uk-keyboard-console.md

```diff
@@ -0,0 +1,63 @@
+---
+title: UK keyboard layout not supported in Web Console | UKCloud Ltd
+description: Intended for customers who are experiencing issues when using a UK keyboard to interact with the VMware Web Console
+services: vmware
+author: Sue Highmoor
+toc_rootlink: Troubleshooting
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: UK keyboard layout not supported in Web Console
+toc_fullpath: Troubleshooting/vmw-ref-trouble-uk-keyboard-console.md
+toc_mdlink: vmw-ref-trouble-uk-keyboard-console.md
+---
+
+# UK keyboard layout not supported in Web Console
+
+## Issue
+
+In vCloud Director 9.1, the VMware Web Console no longer supports UK keyboards. This means that when interacting with the console using a UK keyboard, the keys you press on the keyboard may not work as expected in the console. For example, if you press the @ key on your keyboard, this may appear in the console as a " symbol.
```

Comment:
VMware Web Console has never supported UK Keyboard.

More correct would be:
"The Web Console available in the vCloud Director 9.1 HTML5 UI does not support UK keyboard maps."